website/www/site/content/en/contribute/release-guide.md

```diff
@@ -271,76 +280,12 @@ After following one of these processes you should manually update `CHANGES.md` o
-  1. Change and commit version number in release branch:
-
-     [version.py](https://github.com/apache/beam/blob/release-2.6.0/sdks/python/apache_beam/version.py#L21),
-     [build.gradle](https://github.com/apache/beam/blob/release-2.6.0/runners/google-cloud-dataflow-java/build.gradle#L39),
-     [gradle.properties](https://github.com/apache/beam/blob/release-2.16.0/gradle.properties#L27)
-
-#### (Alternative) Run all steps manually
-* **Checkout working branch**
-
-  Check out the version of the codebase from which you start the release.
-  For a new minor or major release, this may be `HEAD` of the `master` branch.
-  To build a hotfix/incremental release, instead of the `master` branch, use the release tag of the release being patched.
-  (Please make sure your cloned repository is up-to-date before starting.)
-
-      git checkout <master branch OR release tag>
-
-  **NOTE**: If you are doing an incremental/hotfix release (e.g. 2.5.1), please check out the previous release tag, rather than the master branch.
-
-* **Set up environment variables**
-
-  Set up a few environment variables to simplify Maven commands that follow.
-  (We use `bash` Unix syntax in this guide.)
-
-      RELEASE=2.5.0
-      NEXT_VERSION_IN_BASE_BRANCH=2.6.0
-      BRANCH=release-${RELEASE}
-
-  Version represents the release currently underway, while next version specifies the anticipated next version to be released from that branch.
-  Normally, 1.2.0 is followed by 1.3.0, while 1.2.3 is followed by 1.2.4.
-
-  **NOTE**: Only if you are doing an incremental/hotfix release (e.g. 2.5.1), please check out the previous release tag, before running the following instructions:
-
-      BASE_RELEASE=2.5.0
-      RELEASE=2.5.1
-      NEXT_VERSION_IN_BASE_BRANCH=2.6.0
-      git checkout tags/${BASE_RELEASE}
-
-* **Create release branch locally**
-
-      git branch ${BRANCH}
-
-* **Update version files in the master branch**
-
-      # Now change the version in existing gradle files, and Python files
-      sed -i -e "s/'${RELEASE}'/'${NEXT_VERSION_IN_BASE_BRANCH}'/g" build_rules.gradle
-      sed -i -e "s/${RELEASE}/${NEXT_VERSION_IN_BASE_BRANCH}/g" gradle.properties
-      sed -i -e "s/${RELEASE}/${NEXT_VERSION_IN_BASE_BRANCH}/g" sdks/python/apache_beam/version.py
-
-      # Save changes in master branch
-      git add gradle.properties build_rules.gradle sdks/python/apache_beam/version.py
-      git commit -m "Moving to ${NEXT_VERSION_IN_BASE_BRANCH}-SNAPSHOT on master branch."
-
-* **Check out the release branch**
-
-      git checkout ${BRANCH}
-
-* **Update version files in release branch**
-
-      DEV=${RELEASE}.dev
-      sed -i -e "s/${DEV}/${RELEASE}/g" sdks/python/apache_beam/version.py
-      sed -i -e "s/${DEV}/${RELEASE}/g" gradle.properties
-      sed -i -e "s/'beam-master-.*'/'beam-${RELEASE}'/g" runners/google-cloud-dataflow-java/build.gradle
-
+  2. Increment the minor version in the master branch.
```

Comment:
I don't think we need to repeat the information already mentioned above but up to you.